jobbergate-cli/jobbergate_cli/subapps/job_submissions/app.py

```diff
@@ -172,12 +177,14 @@ def list_all(
 @app.command()
 def get_one(
     ctx: typer.Context,
-    id: int = typer.Option(..., "--id", "-i", help="The specific id of the job submission."),
+    id: Optional[int] = typer.Argument(None, help="The specific id of the job submission to be selected."),
+    id_option: Optional[int] = typer.Option(None, "--id", "-i", help="Alternative way to specify id"),
```

Comment:
I guess this is confusing and can lead the user to the `Abort` exception defined in `sanitize_id_selection` unnecessarily. I mean, you could just use `id` instead of both, couldn't you?